src/main/java/com/mattermost/integration/figma/api/figma/notification/service/FileNotificationServiceImpl.java

```diff
@@ -87,16 +90,35 @@ public void subscribeToFileNotification(InputPayload inputPayload) {
         String botAccessToken = inputPayload.getContext().getBotAccessToken();
```

Comment:
please rename  public void subscribeToFileNotification method . Cause inside this method we just trying to create webhook